src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
```

Comment:
```suggestion
    private static final int DEFAULT_CAPACITY = 16; // or INITIAL
    private static final float LOAD_FACTOR = 0.75f;
```
load factor won't ever change in our impl